pkgs/misc/vscode-extensions/list_installed_exts.sh

```diff
@@ -0,0 +1,17 @@
+#! /usr/bin/env bash
+set -eu -o pipefail
+
+# See if we can find our `code` binary somewhere.
+# If the user passed in positional parameters, assume that the first is the
+# path to the code binary. Otherwise try 'code' and 'code-insiders'.
+if [ $# -ne 0 ]; then
+    CODE=$1
+else
+    CODE=$(command -v code || command -v code-insiders)
```

Comment:
please usw which

Reply:
What would you switch on though? The `$#` part? There's doesn't seem to be a single value I can switch on as far as I can see.

Reply:
command also returns exit code 0 when a function named code is defined which which does not.

Reply:
Oh wow sorry I somehow read "switch" instead of "which". So I just kept 90% of the code from the way it was before and there just used `command` without giving it too much thought to be honest

Reply:
which and command shouldn't matter here because my dotfiles shouldn't pollute the update script.